src/bot.js

```diff
@@ -3,138 +3,129 @@ const TelegramBot = require("node-telegram-bot-api");
 const game = require("./game.js");
 const helpers = require("./helpers.js");
 const db = require("./db.js");
+const lang = require("./lang.js");
+let botMessages = {};
```

Comment:
Global variables not allowed

Reply:
Add i18n modul and use it to translate messages